src/components/Game/Wordsy/WordsyData.js

```diff
@@ -57,7 +87,7 @@ function WordsyData() {
       return 'game-over'
     }
 
-    if (currentGuess.length !== languageConfig?.wordLength) {
+    if (currentGuess.length !== WORD_LENGTH) {
       setNotEnoughLettersModalOpen(true)
```

Comment:
One way that you could reduce the length and cognitive load would be by setting the modal content in state rather than maintaining the open state of multiple modals. Just an idea.